target/snitch_cluster/sw/apps/common.mk

```diff
@@ -16,6 +16,11 @@ include $(MK_DIR)/../toolchain.mk
 # Fixed paths in repository tree
 ROOT     := $(abspath $(MK_DIR)/../../../..)
 SNRT_DIR := $(ROOT)/sw/snRuntime
+
+ifeq ($(BIST), ON)
+RISCV_CFLAGS += -DBIST
+endif
+
```

Comment:
Is there an advantage in passing `BIST=ON` on the command-line rather than `RISCV_CFLAGS=-DBIST`? Also, where is this used?

Reply:
No Indeed I will remove it.